released/terraform-be/main.tf

```diff
@@ -0,0 +1,35 @@
+terraform {
+  required_providers {
+    btp = {
+      source  = "sap/btp"
+      version = "1.3.0"
+    }
+    cloudfoundry = {
+      source = "SAP/cloudfoundry"
+      version = "0.1.0-beta"
+    }
+  }
+}
+
+provider "btp" {
+  globalaccount  = "terraformintcanary"
+  cli_server_url = "https://canary.cli.btp.int.sap"
+}
+
+###############################################################################################
+# Setup of names in accordance to naming convention
+###############################################################################################
+locals {
+  random_uuid               = uuid()
```

Comment:
This is dangerous as the `uuid()` function is called every time the script gets executed. It is not bound to the lifecycle and will result in diffs although there are none 

You should use the [random_uuid resource](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/uuid) instead